docs/en/observability/exploratory-data-visualizations.asciidoc

```diff
@@ -1,28 +1,201 @@
+* Synthetics monitoring
+* User experience (RUM)
+* Infra metrics
+* Mobile experience
+. Click *Select report metric* and select the options and filters you need. You
+will see a *Missing...* warning if required fields (highlighted with red
+  underline) are incomplete.
+. Click *Apply changes* to see the updated visualization or repeat the *Add series* process to expand the visualization.
+. To add the visualization to an existing case, click *Add to case* and select the correct case.
+
+[discrete]
+[[metrics]]
+== Metrics
+
+Based on your infrastructure metrics, you can create various visualizations relating to
+memory or CPU usage, or any of the <<data-types>>[available data types].
+
+[role="screenshot"]
+image::images/exploratory-view-metrics.png[Explore data for Infra metrics]
+
+|===
+
+| *KPI over time* | The KPI over time histogram represents the performance indicators based on
+the metric you select.
+
+By default, the `system CPU usage` metric is selected.
+
+|===
+
+[discrete]
+[[explore-data-synthetics]]
+== Synthetic Monitoring
+
+Based on the Uptime data you are sending to your deployment, you can create various visualizations relating to
+monitor durations, pings over time, or any of the <<data-types>>[available data types].
+
+[role="screenshot"]
+image::images/exploratory-view-uptime.png[Explore data for Uptime]
+
+|===
+
+| *Monitor duration* | The Uptime monitor duration time-series chart displays the timing for each check that {heartbeat} performed.
+
+This visualization helps you to gain insights into how quickly requests resolve by the targeted endpoint
+and give you a sense of how frequently a host or endpoint was down in your selected timespan.
+
+| *Pings histogram* | The Uptime pings chart is a graphical representation of the check statuses over time.
+
+|===
+
+[discrete]
+[[explore-data-user-experience]]
+== User Experience
+
+Based on the {user-experience} data from your instrumented applications, you can create
+detailed visualizations for performance distributions, key performance indicators (KPI) over time,
+and for core web vitals of your web applications.
+
+[role="screenshot"]
+image::images/exploratory-view-ux-page-load-time.png[Explore data for User Experience - page load time]
+
+|===
+
+| *KPI over time* | The KPI over time histogram represents the performance indicators based on
+the metric you select.
+
+By default, the `page views` metric is selected.
+
+| *Performance distribution* | The Performance distribution time-series chart enables you to examine the perceived
+performance of your web applications based on the metric you select.
+
+By default, the `page load time` metric is selected.
+
+| *Core web vitals* | The Core web vitals chart is a graphical representation of key metrics, such as
+loading performance, load responsiveness, and visual stability, for each of your web applications.
+
+By default, the `largest contentful paint` metric is selected. Hover over the chart to display crosshairs
```

Comment:
Earlier you use init cap rather than formatting with backticks. I would be consistent.